test/product-search-test.js

```diff
@@ -59,6 +59,98 @@ describe('Product Search', function() {
 
   });
 
+  it('summary should return appropriate products from store/s or brand/s 1', function(done){
```

Comment:
@bethleha Can we not qualify these test cases with numbers but instead add a more elaborate description?